addon/core/keymap.ts

```diff
@@ -79,7 +94,7 @@ const del = chainCommands(
 /// * **Delete** and **Mod-Delete** to `deleteSelection`, `joinForward`, `selectNodeForward`
 /// * **Mod-Delete** to `deleteSelection`, `joinForward`, `selectNodeForward`
 /// * **Mod-a** to `selectAll`
-export const pcBaseKeymap: Keymap = (schema: Schema) => ({
+export const pcBaseKeymap: Keymap = (schema, options) => ({
```

Comment:
its ok for now, I suspect we'll revisit the outside-facing interface when we actually turn this on